src/userSurvey.js

```diff
@@ -122,9 +129,30 @@ function setPopupToNoPromptPage() {
     });
 }
 
+/**
+ * Initialize storage for the module.
+ * @private
+ */
+function initializeStorage() {
+    if (storageSpace === null) {
+        storageSpace = storage.createKeyValueStorage("webScience.userSurvey");
+    }
+}
+
+/**
+ * Called when the current survey is completed. Sets surveyCompleted to true in storage
+ * and changes the browser action popup to the survey's no prompt page.
+ * @private
+ */
+function setSurveyComplete() {
+    storageSpace.set("surveyCompleted", true);
+    setPopupToNoPromptPage();
+}
+
```

Comment:
Would use a different function name, to avoid confusion with the similar `setSurvey`.